src/lib/cryptography/login.ts

```diff
@@ -30,6 +31,9 @@ export const decryptNativeWallet = async (
     const encryptedWallet = getEncryptedWallet()
     await encryptedWallet.init(email, password, encryptedWalletString)
     const mnemonic = await encryptedWallet.decrypt()
+    if (!bip39.validateMnemonic(mnemonic)) {
+      throw new Error('Invalid Mnemonic')
+    }
```

Comment:
Don't think this is necessary. There's a MAC on the encrypted wallet and decryption will fail instead of giving invalid data.
```suggestion
```

Reply:
I agree, it seemed superfluous given the way we decrypt.

Removed.